.gitignore

```
@@ -159,6 +159,7 @@ fv3-jedi-data/
 fv3-jedi-lm/
 gsibec/
 gsw/
+icepack
```

guessing you need a trailing /
```suggestion
icepack/
```

---

it works without the ```/``` , but sure.